TTPChallenge/QuizMainVC.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  QuizMainVC.swift
+//  TTPChallenge
+//
+//  Created by Luna An on 2/14/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class QuizMainVC: UIViewController {
+
+    @IBOutlet weak var startButton: UIButton!
+    @IBOutlet weak var partOneCircle: UILabel!
+    @IBOutlet weak var partTwoCircle: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Styling elements
+        startButton.layer.cornerRadius = 20
+        partOneCircle.layer.cornerRadius = partOneCircle.frame.width/2
+        partOneCircle.layer.masksToBounds = true
+        partTwoCircle.layer.cornerRadius = partTwoCircle.frame.width/2
+        partTwoCircle.layer.masksToBounds = true
+        
```

Comment:
see above on how to do this in storyboard. the maskToBounds would be the same. the key would be layer.maksToBounds, the type would be Bool and the value would be true or false